assets/src/stories-editor/helpers/addAMPAttributes.js

```diff
@@ -194,6 +194,7 @@ const addAMPAttributes = ( settings, name ) => {
 		supports: {
 			...settings.supports,
 			anchor: false,
+			align: false,
```

Comment:
Would this set `align:false` to all the blocks, including `table`, `verse`, etc.?

Reply:
Don't we want to remove alignment for all blocks? If not, I can easily just remove support conditionally.

Reply:
Just checked and the table block does support alignments but they don't work. 
If we are going to support alignments then that is a bigger issue that we should look at.

Reply:
Hmm, based on what I can see then #3504 mentions specific blocks only (Image, pullquote, embed, video). Probably some of the blocks, e.g. Verse, List, possibly Table, should still have the option for alignment. Although now when testing, it looks like the `align:false` doesn't influence those blocks either, it looks like "Verse" still has alignment despite setting it to `false`. I guess that's also the Gutenberg bug? Probably should still make it conditional since when the bug gets fixed this would remove the support for the blocks that need alignment, too.

If the Gutenberg bug would get fixed, would this remove the alignment from the Text block, too with this?

It feels "more correct" to specifically remove the alignment from the blocks that don't need it instead of all. Thoughts?

(Note: I can see the alignment for Table working):
<img width="334" alt="Screenshot 2019-10-28 at 18 38 49" src="https://user-images.githubusercontent.com/3294597/67702550-34646380-f9b2-11e9-8ec4-8ef3d09940f5.png">

Reply:
@swissspidy Can you confirm what the action should be here? I believe we should not support alignment for any block. In my tests, only right, center and left kind of worked for table, but wide and full did not. We either support property or not at all.

Reply:
Note that there is also `alignWide: false` option.

Also, in case of Table, the full width does work, e.g. compare left alignment with full width:
<img width="286" alt="Screenshot 2019-10-28 at 19 42 18" src="https://user-images.githubusercontent.com/3294597/67707642-236c2000-f9bb-11e9-872c-4710a8bd9f5f.png">
<img width="302" alt="Screenshot 2019-10-28 at 19 42 11" src="https://user-images.githubusercontent.com/3294597/67707644-236c2000-f9bb-11e9-912c-73b675c92351.png">

Reply:
Table block has alignment options for the Table generally and separately for the Text within. Probably alignment for the text inside makes sense, for the Table not (since resizing replaces that).

Reply:
> Table block has alignment options for the Table generally and separately for the Text within. Probably alignment for the text inside makes sense, for the Table not (since resizing replaces that).

That's my thinking as well.

We have draggable and resizable blocks; any alignment options that target the  _block_ and not just its _text_ are not useful and should be disabled.

Reply:
To answer the question:

> Don't we want to remove alignment for all blocks?

Yes, we do. For all blocks.

To give an example:

Remove block aligments:

![Screenshot 2019-10-29 at 11 49 29](https://user-images.githubusercontent.com/841956/67760808-48f03c80-fa42-11e9-8e94-36204180f9ec.png)

Keep text or column alignments:

![Screenshot 2019-10-29 at 11 49 24](https://user-images.githubusercontent.com/841956/67760823-4e4d8700-fa42-11e9-90c0-516c1931260a.png)

Reply:
So the PR as it stands is correct. `align: false` remove the block alignment and not the column / text alignment. With alignment disabled `alignWide: true` does nothing.